src/cdk/drag-drop/drag-ref.ts

```diff
@@ -744,6 +753,15 @@ export class DragRef<T = any> {
       this._scrollPosition = this._viewportRuler.getViewportScrollPosition();
     });
 
+    // If the page is blurred while dragging (e.g. there was an `alert` or the browser window was
+    // minimized) we won't get a mouseup/touchend so we need to use a different event to stop the
+    // drag sequence. Use the last known location to figure out where the element should be dropped.
+    this._blurSubscription = this._dragDropRegistry.pageBlurred.subscribe(() => {
```

Comment:
I would have thought that this would be treated as cancelling the drag sequence instead of completing it, since the dragged item might be somewhere the user doesn't want to drop.

Reply:
We could do that, but it won't be very smooth, because the list could've been shuffled already and we'd have to snap it back into place. Also it would be inconsistent, because we'd have to avoid emitting the `dropped` event, even though we've emitted the `start`, `move` etc. events.

Reply:
Do users currently have a way to cancel a drag? E.g., is I want to let people press escape to abort?

Reply:
They don't. I've had it as a low-priority task on my list to add an API for it, but I haven't had the time for it.

Reply:
I think I would want to collect more data on this before committing to the behavior (drop vs. abort)- maybe trying it out with a few people and seeing what they expect to happen

Reply:
Do we have people that are willing to try it out? This is a bit of an edge case to begin with so I don't know how much time we want to spend on it.

Reply:
Maybe just a twitter poll?